src/Modal/Modal.tsx

```diff
@@ -82,7 +82,7 @@ interface Props {
   /**
    * Title of the modal
    */
-  title: React.ReactNode;
+  title?: React.ReactNode;
```

Comment:
this could have been passed as null before, and there is a null check on https://github.com/apollographql/space-kit/pull/373/files#diff-1be7e63e7570ebe66e4559909557f52a74c7eaef41906bb9e6078e0d376201c3R263